styles.css

```diff
@@ -7,27 +7,30 @@ body {
 
 @import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
 
+* {
+  font-family: "Poppins", sans-serif;
+}
+
 @media (min-width: 375px) {
```

Comment:
- [ ] Since we are using `mobile first` approach, I will suggest you not use media query (`@media`) for now. You can use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels to make your page responsible .